pkgs/development/libraries/gettext/default.nix

```diff
@@ -47,7 +48,7 @@ stdenv.mkDerivation rec {
     sed -i -e "s/\(libgettextsrc_la_LDFLAGS = \)/\\1..\/gnulib-lib\/libxml_rpl.la /" gettext-tools/src/Makefile.in
   '';
 
-  nativeBuildInputs = [ xz xz.bin bison automake115x autoconf];
+  nativeBuildInputs = [ xz xz.bin automake115x autoconf];
```

Comment:
Here we would need to only have `bison` is we are not on darwin, again, to have no rebuild. Also could you, please, add comment that these three deps are temporary and only needed for CVE patch.

Reply:
Done in Done in 9bc0d0efb7c11027139de24c1c415e694757bc8b..